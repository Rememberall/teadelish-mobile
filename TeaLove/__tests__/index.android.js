import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import Index from '../index.android';

it('renders correctly', () => {
  const tree = renderer.create(
    <Index />,
  );

  expect(tree).toMatchSnapshot();
});
