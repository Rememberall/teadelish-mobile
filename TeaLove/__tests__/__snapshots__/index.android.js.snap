exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }>
    Welcome to React Native!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    To get started, edit index.android.js
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    Double tap R on your keyboard to reload,
    
    
    Shake or press menu button for dev menu
  </Text>
</View>
`;
